react/features/filmstrip/components/native/Thumbnail.js

```diff
@@ -283,7 +283,6 @@ function _mapStateToProps(state, ownProps) {
     return {
         _audioTrack: audioTrack,
         _isEveryoneModerator: isEveryoneModerator(state),
-        _isModerator: isLocalParticipantModerator(state),
```

Comment:
_isModerator was no longer used?

Reply:
it was used at line 159, and here I'm not sure why is this change being made ? Is the remote menu for every user now and being moderated in the component ? a change like that would fit into a separate commit with explanation

Reply:
oohh so now you're changing each button to decide by itself. Im not sure if it's a good thing. What if all buttons will decide to not show up and empty menu would be displayed ?

Reply:
I don't think the buttons should be deciding. They seem to not be in the right position to make such decisions, but it's just an opinion so feel free to decide by yourself. I feel in long term this may cause problems at some point and require to be refactored back to where it is now.

Reply:
No, because we need to bring up the remote video menu even when the local participant is not a moderator (to be able to set a participant on stage). Now we can ask if opening the menu when not in tile view makes any sense, but my thinking was that it would probably involve some heavy logic to decide if there are any menu items to show, and popping up the display name is not a bad thing after all, but definitely not a bug. Opinion?

Reply:
Sorry, I didn't see Pawels answers before answering.

@paweldomas your thinking is right, but that is just one way of putting it. Here is my thinking: A button should be able to decide if it's visible or not, that's how it becomes fully portable. E.g. if we want to relocate a menu item from the remote video menu into the "..." menu, we just move the component, and it's done. It never gets falsly rendered, because the component knows for itself if he has the right to be shown, or not.

then about the question of empty menu: yes, that may be an issue in another situation, but not now. This menu always have an element, it's the display name, that was an original request from Emil to be able to find out the display name of a participant by long pressing the thumbnail.

I guess it's just a collision of approaches, both have pros and cons

Reply:
I didn't know about the display name. Right, those buttons are specific to the menu feature, so no real problem with that. It's just two different approaches and this one will work for now.